Update test (#1)
fitbody/jr-frontend-challenge: README.md

# Rick and Morty App!

The main objective of this challenge is to test your abilities as a front-end developer.

## Challenge

Create a web application that shows every **Rick and Morty** character with its description.

> > **Note:** For this challenge you'll have to use ReactJs or VueJs. You can also use design libraries such as [Ant Design](https://ant.design), [Material-UI](https://material-ui.com), [React bootstrap](https://react-bootstrap.github.io), among others.

Reviewer: ```suggestion
> **Note:** For this challenge you must use ReactJs or VueJs. Also you can use design libraries such as [Ant Design](https://ant.design), [Material-UI](https://material-ui.com), [React bootstrap](https://react-bootstrap.github.io), among others.
```

### Instructions

> You'll have to consume the following REST API: [The Rick and Morty API](https://rickandmortyapi.com)

Reviewer: ```suggestion
You must consume the following REST API: [The Rick and Morty API](https://rickandmortyapi.com)
```
